.github/workflows/ci.yml

```diff
@@ -2,25 +2,48 @@ name: Python CI
 
 on:
   push:
-    branches: [ main, dev ]
+    branches: 
+      - main
+      - dev
+      - 'feature/**'
   pull_request:
-    branches: [ main, dev ]
+    branches: 
+      - main
+      - dev
+      - 'feature/**'
```

Comment:
Currently we're not using any branch naming convention (except for me 😉 I typically use feat, chore, doc, etc.). 
So should everyone prepend their branches with `feature/` in order to trigger this?

Reply:
yes please! but if you have any preferences I can change it